src/metabase/api/field.clj

```diff
@@ -408,6 +397,10 @@
 
       [<value-of-field> <matching-value-of-search-field>].
 
+   If `search-field` and `field` are the same, simply return 1-vectors like
```

Comment:
to confirm: this is how FE understands it too right?

Reply:
Yes. Here's the type used by the `FieldValuesWidget` for example:
https://github.com/metabase/metabase/blob/567cdb7fe62a59a11ae720bae12c4abbdd4a07aa/frontend/src/metabase-types/api/field.ts#L53